doc/nrf/ug_bootloader.rst

```diff
@@ -92,7 +92,96 @@ These signed variants can be used to perform an upgrade procedure through the :r
 Adding a bootloader chain to your application
 *********************************************
 
-See :ref:`bootloader_build_and_run` in the documentation of the :ref:`bootloader` sample for instructions on how to add an immutable bootloader as first stage of the secure bootloader chain.
+The |NCS| includes a sample implementation of an immutable bootloader.
+As upgradable bootloader, it provides and extends :doc:`mcuboot:index`.
 
-See :doc:`mcuboot:index` for information on how to implement an upgradable bootlader.
+Both bootloaders can easily be included in your application using :ref:`ug_multi_image`.
+
+Adding the immutable bootloader
+==============================
+
+To add an immutable bootloader to your application, set :option:`CONFIG_SECURE_BOOT` and add your private key file under :option:`CONFIG_SB_SIGNING_PUBLIC_KEY`.
+|how_to_configure|
+
+See the documentation of the :ref:`bootloader` sample for more information.
+The :ref:`bootloader_build_and_run` section has detailed instructions for adding the immutable bootloader as first stage of the secure bootloader chain.
+
+Adding MCUboot as an upgradable bootloader
+===============================
```

Comment:
```suggestion
==========================================
```
No idea if I got the right amount of = now - needs to be the same length as the heading. ;)